### ISSUE TITLE
Send email/text when the user is the nth in line (n chosen by users)

### ISSUE THREAD

**Issue by Zackhardtoname** [mterwill/office-hours-help-queue#149]
So that the users don't have to constantly check for their spot.
A quick temporary solution might be to call the JS alert function to make an alert?

**Comment by mterwill** [comment]
👋 there is already a feature that will send a browser push notification when you're next up. Would that serve your needs?

A panel like this should show up on the queue page if you've not already dismissed it:

![image](https://user-images.githubusercontent.com/5882053/76547101-06645500-6463-11ea-83d4-886591c8f617.png)

**Comment by Zackhardtoname** [comment]
Thank you for pointing that out. I actually disabled all notifications by default in Chrome (Version 80.0.3987.132) so I don't think I got the panel (or I missed it...).
Anyhow, after enabling it for the site, it worked well. 

I still think there's value in text/email notifications just in case the user has to leave the laptop to find a charging outlet or go to the bathroom.

**Comment by Prajna-Ty** [comment]
I agree. That feature would be very helpful. I went to refill my waterbottle yesterday, and by the time when I was back, I was no longer on the queue... :(

**Comment by lmcicat** [comment]
Hey @mterwill ! @brianguo98 and I need to make open source contributions for our final homework assignment for a class this semester. We were hoping to try to tackle this issue if that is ok with you?

**Comment by mterwill** [comment]
👋 Hey @lmcicat and @brianguo98, that's awesome! Happy & appreciative to have y'alls contribution.

Before you get started, would you mind dropping an outline of how you anticipate the technical implementation of this feature will look and the various components involved?

**Comment by lmcicat** [comment]
Hey @mterwill , sorry for the delay in response. We unfortunately weren't approved by course staff to work on this repo :(. Once the semester ends, I may come back here and try to tackle this when I have more free time!

**Comment by mterwill** [comment]
Bummer! Thanks for the heads up.

**Comment by annieli555** [comment]
Hi again! @pl98 and I are also interested in working on this feature request. Is this request still standing?

**Comment by mterwill** [comment]
I think this one will be tricky, requiring some infrastructure work as well. Same request as above, if you could outline what you’re thinking and drop it here before starting on any implementation, that’d be great.
